### ISSUE TITLE
Wild-type genotype edit restriction is unhelpful for PHI-Canto

### ISSUE THREAD

**Issue by jseager7** [pombase/canto#1842]
(Possibly related: #1244)

@CuzickA is requesting a change to the logic in pathogen–host mode when editing wild-type genotypes. Currently, Canto forbids finishing the edit, and shows a warning:

> A wild type allele with wild type product level is only valid in a genotype that has another allele of the same gene.

(See below for a screenshot.)

According to @CuzickA, this isn't ideal for PHI-Canto because "we need to be able to curate wild type control interactions for PHI-base." I don't understand the subtleties here, so this might need some advice from @ValWood or another curator.

Note that there is a workaround that involves using Copy and Edit on the genotype instead, probably to trick Canto into thinking that another allele of the same gene exists, but this definitely isn't intuitive.

The fix might be as simple as disabling the restriction in pathogen–host mode, but I'm not sure since I haven't investigated the code that decides whether the warning should be displayed.

![wt_edit_warning](https://user-images.githubusercontent.com/37659591/56809875-cd13fe00-682c-11e9-8d63-74a9f21a94be.png)

**Comment by kimrutherford** [comment]
We can add a new option to disable the wildtype expression checking.  I'll do that on Monday.

**Comment by kimrutherford** [comment]
We discussed this issue on yesterday's call.  We decided that it would make sense for Alayne and Val to have a chat about this at some point to make sure that this change will allow Alayne to capture the information she needs to.

In the meantime, it was a quick fix so I've gone ahead and added an extra configuration setting that allows the wildtype allele checks to be disabled.

@jseager7 you'll need to add this to the PHI-Canto config file to test the change:
```
allow_single_wildtype_allele: 1
```
